airflow-core/src/airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -619,6 +620,7 @@ def delete_asset_queued_events(
 @assets_router.delete(
     "/dags/{dag_id}/assets/queuedEvents",
     status_code=status.HTTP_204_NO_CONTENT,
+    response_model=None,
```

Comment:
I don’t think we need these since the PR already adds `-> None` to those functions. This parameter should only only needed if you return something different from the return type annotation.

https://fastapi.tiangolo.com/tutorial/response-model/